model/page.go

```diff
@@ -125,7 +125,7 @@ func (r *PdfReader) newPdfPageFromDict(p *core.PdfObjectDictionary) (*PdfPage, e
 		page.LastModified = &lastmod
 	}
 
-	if obj := d.Get("Resources"); obj != nil {
+	if obj := d.Get("Resources"); obj != nil && !core.IsNullObject(core.ResolveReference(obj)) {
```

Comment:
Would it make sense that core.IsNullObject resolves references if it encounters those there?
Would be slightly cleaner

Reply:
Yes, that makes sense. I made the changes.